change/@internal-react-composites-50ba02c6-2f36-4307-85c5-a1adc26cd271.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
This should be none as it doesn't impact any external users of the npm package

Reply:
Done